earth_enterprise/src/fusion/autoingest/plugins.src/RasterProject.src

```diff
@@ -1153,16 +1155,20 @@ uint32 RasterProjectAssetVersionImplD::BuildIndex(
       genInfo[i].PopulatePacketLevels(packgen);
     }
 
-    // remember this new PacketGen version
-    packgenkids.push_back(packgen->GetRef());
-    packgenassets.push_back(packgen->GetAssetRef());
-    packgenvers.push_back(packgen);
+#pragma omp ordered 
+    {
+      // remember this new PacketGen version
+      packgenkids.push_back(packgen->GetRef());
+      packgenassets.push_back(packgen->GetAssetRef());
+      packgenvers.push_back(packgen);
+    }
   }
 
   curTime = std::chrono::high_resolution_clock::now();
   timeDiff = std::chrono::duration<double>(curTime - prevTime);
   notify(NFY_INFO2, "BuildIndex finished looping through insets - %f seconds elapsed",
          timeDiff.count());
+  assert(MiscConfig::Instance().ContLoadAndThink);
```

Comment:
I added this to cut off large builds at the same point.  When fusion is killed here, nothing is written out to file yet so there is no need to clean up or redo setup for the next build.  I also use this to help get metrics on how long a specific build took.  Definitely don't plan on keeping this.